generator-mediator-js/README.md

```diff
@@ -1,45 +1,37 @@
 # generator-mediator-js
 
-> [Yeoman](http://yeoman.io) generator
-
 
 ## Getting Started
 
-### What is Yeoman?
-
-Trick question. It's not a thing. It's this guy:
-
-![](http://i.imgur.com/JHaAlBJ.png)
-
-Basically, he wears a top hat, lives in your computer, and waits for you to tell him what kind of application you wish to create.
-
-Not every new computer comes with a Yeoman pre-installed. He lives in the [npm](https://npmjs.org) package repository. You only have to ask for him once, then he packs up and moves into your hard drive. *Make sure you clean up, he likes new and shiny things.*
+### How to use our friend Yeoman to generate a new mediator
 
 ```bash
 npm install -g yo
 ```
 
-### Yeoman Generators
-
-Yeoman travels light. He didn't pack any generators when he moved in. You can think of a generator like a plug-in. You get to choose what type of application you wish to create, such as a Backbone application or even a Chrome extension.
-
 To install generator-mediator-js from npm, run:
 
 ```bash
 npm install -g generator-mediator-js
 ```
 
-Finally, initiate the generator:
+Or for dev purposes, from the generator-mediator-js directory, run:
 
 ```bash
-yo mediator-js
+sudo npm link
 ```
 
-### Getting To Know Yeoman
+Finally, from a clean directory into which you would like to install the mediator:
 
-Yeoman has a heart of gold. He's a person with feelings and opinions, but he's very easy to work with. If you think he's too opinionated, he can be easily convinced.
+```bash
+yo mediator-js
+```
 
-If you'd like to get to know Yeoman better and meet some of his friends, [Grunt](http://gruntjs.com) and [Bower](http://bower.io), check out the complete [Getting Started Guide](https://github.com/yeoman/yeoman/wiki/Getting-Started).
+You will be prompted to enter the config needed to create the mediator, to use default config:
+
+```bash
+yo mediator-js --useDefaults --name="Mediator Name" --description="This is what my mediator does"
+```
 
 
 ## License
```

Comment:
I know this was here already, but I think let's remove the license field. Tbh I'm not too sure about how licensing works for something like, but if anything it should be MPL 2.0 (but let's leave it out for now)